external/dockerfile_functions.sh

```diff
@@ -139,7 +139,11 @@ print_debianslim_pkg() {
     local file=$1
     local packages=$2
 
-    print_ubuntu_pkg ${file} "${packages}"
+    echo -e "RUN apt-get update \\" \
```

Comment:
Wondering if we should put a comment in here linking to the issue, so we will have some reminder to un-do this change when the core issue is fixed?

Reply:
just added the comment. Linked to the issue.